client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -74,16 +83,26 @@ export default function UserOperatorMultiStepForm({
       return;
     }
 
-    push(
-      `/dashboard/select-operator/user-operator/create/${
-        formSection + 2
-      }?user-operator-id=${response.user_operator_id}`,
+    // continue to use user-operator-id search param for create route since database will error if sending a user operator id
+    // in the /[id] route as the user doesn't exist yet
+    return push(
+      `/dashboard/select-operator/user-operator/${
+        isCreate ? "create" : userOperatorId
+      }/${formSection + 1}${
+        isCreate ? `?user-operator-id=${response.user_operator_id}` : ""
+      }`,
```

Comment:
@BCerki I mentioned above that I would tag you in this. I had refactored the form to work similar to the `Operations` form with `/[ID]/[formSection]` though due to differences in how these work on the api side I still needed to keep the search param for sending to the page 2 create route.

The api would return an error if passing it to the `/[ID]` route as the user data did not exist yet. There was a lot of weird little issues like that which is why this PR took me awhile.